frontend/src/main/components/Users/UsersTable.js

```diff
@@ -112,6 +112,16 @@ export default function UsersTable({ users}) {
 
     //const columnsToDisplay = showButtons ? buttonColumn : columns;
```

Comment:
Can we remove this comment out line? 

I know it isn't from you, but when we see these in files we touch, it's good to get rid of them.

Reply:
See also: 

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed